config/warren.yml

```diff
@@ -0,0 +1,31 @@
+# By default the development environment just logs the message and
+# payload. If you wish to enable broadcasting in development mode,
+# the easiest way to do so is to set the ENV WARREN_TYPE.
+# For example
+# `WARREN_TYPE=broadcast bundle exec rails s`
+# This will override the setting in warren.yml
```

Comment:
Not sure I quite understand this - previously we changed the `type`on line 8 (or equivalent of) to `broadcast`, in order to get it to broadcast in development mode. Looks like that's the default now. What does `WARREN_TYPE=broadcast` override? Something in the warren gem? 

Thanks

Reply:
Ahh, log is still the default when this file gets generated by warren, but I changed it to broadcast here, as it doesn't really make sense to run the event warehouse in the absence of RabbitMQ. I just forgot to change the original help comment.

Reply:
Updated the comment